File: Programmers/Lv.0/전국 대회 선발 고사/181851.java
import java.util.Map;
import java.util.HashMap;
import java.util.Arrays;

class Solution {
    public int solution(int[] rank, boolean[] attendance) {
        int answer = 0;
        
        Map<Integer, Integer> idxMap = new HashMap<Integer, Integer>();
        int cnt = 0;
        
        for (int i=0; i<rank.length; i++) {
            if (attendance[i]) {
                idxMap.put(rank[i], i);
                cnt++;
            } else {
                rank[i] = 999;
            }
        }
        
        Arrays.sort(rank);
        
        int a = idxMap.get(rank[0]);
        int b = idxMap.get(rank[1]);
        int c = idxMap.get(rank[2]);
        
        answer = (10000 * a) + (100 * b) + c;
        
        return answer;
    }
}

File: Programmers/Lv.0/전국 대회 선발 고사/181851.js
function solution(rank, attendance) {
    let answer = 0;
    
    const arr = [];
    for (let i=0; i<rank.length; i++) {
        if (attendance[i]) {
            arr.push([rank[i], i]);
        }
    }
    
    arr.sort((a, b) => a[0] - b[0]);
    
    const a = arr[0][1];
    const b = arr[1][1];
    const c = arr[2][1];
    
    answer = (10000 * a) + (100 * b) + c;
    
    return answer;
}
